ts/boot/configurePushNotification.ts

```diff
@@ -29,6 +40,28 @@ const NotificationPayload = t.partial({
   })
 });
 
+/**
+ * Decide how to refresh the messages based on pagination.
+ */
+function handleMessageReload() {
+  const cursors = getCursors(store.getState());
+  if (pot.isNone(cursors)) {
+    // nothing in the collection, refresh
+    store.dispatch(reloadAllMessages.request({ pageSize }));
+  } else if (pot.isSome(cursors)) {
+    // something in the collection, get the new ones only
```

Comment:
if I well understood, we are trying to load the most large page size as possible to have an high probability that the new message is included in that page. Is it right? if it is so, could you extend the comment by describing this?

Reply:
Good point, updated [here](https://github.com/pagopa/io-app/pull/3580/commits/5483199366753b094a6034819992105e6a4769ad)